pygenesys/tests/test_inp/test_inputfile.py

```diff
@@ -0,0 +1,24 @@
+"""
+PyGenesys Input File
+To run execute:
+
+```bash
+genesys --infile my/pygenesys/input/file.py
+```
+"""
+# So the database can be saved in the location from which
+# the command is called.
+import os
+curr_dir = os.path.dirname(__file__)
+
+database_filename = 'my_temoadb.sqlite'  # where the database will be written
+scenario_name = 'test'
+start_year = 2025
+end_year = 2050
+year_step = 5
+N_seasons = 4  # the number of seasons in the model
+N_hours = 24  # the number of hours in a day
+"""
+Note: If the (end_year - start_year) % year_step != 0, then
+the final year of the simulation will be truncated.
```

Comment:
This is a good thing to add to your tests when you test the years. A couple of scenarios where `% year_step = 0` and `% year_step != 0` would be good to check.

Reply:
I changed the input file to only accept a number of years rather than a step size. This change sidesteps the problem. The first and last years specified will _always_ be in the simulation.
This change is coming in the next PR